packages/core/src/components/banner/banner.scss

```diff
@@ -0,0 +1,16 @@
+:host {
+  display: block;
+
+  --di-status-success: #{dso-icon("status-success")};
+  --di-status-danger: #{dso-icon("status-danger")};
+  --di-status-warning: #{dso-icon("status-warning")};
+  --di-status-info: #{dso-icon("status-info")};
```

Comment:
Ik zie elders dat banner maar twee statussen kent (danger en warning), waarom staan hier nog twee extra varianten?

Reply:
En waar worden deze variablen eigenlijk gebruikt?

Reply:
Goed gezien, `success` en `info` moeten weg. Als ik zoek op deze CSS variabelen krijg ik geen resultaat terug, maar als ik deze verwijder dan verdwijnt het icoon in het voorbeeld. Wordt dit misschien in jou dso-icon script afgevangen @tfrijsewijk?

Reply:
Ik zie het al: https://github.com/dso-toolkit/dso-toolkit/pull/1006/files#diff-cb74234a4842fbb6d05d833f77509d2c1a56564530b15fe0d6d6939f7c3b2a77R106

Wel heel suf dat we die modifier classes met `alert-` hebben laten prefixen ipb met `banner-`..